src/closer.rs

```diff
@@ -29,30 +29,31 @@ impl Closer {
 
 #[cfg(test)]
 mod tests {
-    use yatp::task::callback::Handle;
-
     use super::*;
 
     #[test]
     fn test_closer() {
-        let pool = yatp::Builder::new("test_closer").build_callback_pool();
         let closer = Closer::new();
 
         let (tx, rx) = unbounded::<()>();
 
+        let mut handles = vec![];
+
         for _i in 0..10 {
             let closer = closer.clone();
             let tx = tx.clone();
-            pool.spawn(move |_: &mut Handle<'_>| {
+            handles.push(std::thread::spawn(move || {
```

Comment:
Does this change is want to workaround the thread sanitizer? If so, master has fix it.

Reply:
I will change it back.